packages/plugin-ext/src/hosted/browser/plugin-worker.ts

```diff
@@ -16,26 +16,41 @@
 import { injectable } from '@theia/core/shared/inversify';
```

Comment:
What's the reason for moving this file?

Reply:
I think it's reasonable to have this file next to `hosted-plugin.ts`, where it's used.
I have a feeling, that before this file was in a wrong place.